cache/redisOps.js

```diff
@@ -182,17 +182,17 @@ function deleteSampleKeys(associationType, name) {
   })
   .then((samples) => {
     deletedSamples = samples || [];
-    cmds = [];
 
-    // remove the entries from the master list of index
-    cmds.push(['srem', indexName, Array.from(keyArr)]);
-
-    // delete the hashes too
-    cmds.push(['del', Array.from(keyArr)]);
-    return redisClient.batch(cmds).execAsync();
+    /*
+     * Delete all the sample hashes and remove the entries from the master
+     * index of samples.
+     */
+    const sampleRemovalCmds = keyArr.map((key) => ['del', key]);
```

Comment:
Any idea why this wasn't working? Redis docs say del can accept multiple keys... Maybe a bug in the npm redis client? Maybe something to do with running the commands as a batch?

Could there be other places in the code where we are doing the same thing that need to be changed?

Reply:
> Redis docs say del can accept multiple keys
Yes, and I've seen it work with a limited number of keys. I wonder if the size of the array matters? I haven't tried testing how many keys tips it over to failure, and I couldn't find any documented limits on the size of the array of keys in the redis or node_redis docs.

> ...other places in the code...
Yeah, I'll look into that.

Reply:
I updated one more place where we do this. The other place we do this in sampleStoreInit, which is deprecated and just waiting to be removed from codebase.